src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -274,18 +274,32 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
 
       <div id="chatBox" class="rounded">
         <div id="messagesContainer">
-          <ul>
-            <%
-              for (Message message : messages) {
-                String author = UserStore.getInstance().getUser(message.getAuthorId()).getName();
-            %>
-                <li> <strong> <a href="/profile/<%=author %>"><%= author %></a>: </strong>
-                <%= message.getContent() %> </li>
-
-            <%
-                }
-            %>
-          </ul>
+          <%
+            for (Message message : messages) {
+              String author = UserStore.getInstance().getUser(message.getAuthorId()).getName();
+              if(user.equals(author)){
+          %>
+                <div class="messageContainer right">
+                  <div class="box rightBubble blue">
+                    <%= message.getContent() %>
```

Comment:
nit: store this is a variable.

Reply:
Not sure what you mean. I don't think I use this anywhere else.

Reply:
I was referring to the message content, but I forgot the message was in an `if-else`. Nvm. 😃